charts/incubator/firezone/values.yaml

```diff
@@ -15,26 +15,85 @@ securityContext:
         - NET_ADMIN
         - SYS_MODULE
 
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        port: 13000
+  wireguard:
+    enabled: true
+    ports:
+      wireguard:
+        enabled: true
+        protocol: udp
+        port: 51820
+
+firezone:
+  web:
+    external_url: "https://example.com"
+    trusted_proxies: []
+    private_clients: []
+  admin:
+    reset_admin_on_boot: false
+    default_email: "admin@email.com"
+    default_password: "1234567890"
+  devices:
+    allow_unprivileged_device_management: true
+    allow_unprivileged_device_config: true
+    vpn_session_duration: 0
+    client_persistent_keepalive: 25
+    default_client_mtu: 1280
+    client_endpoint: ""
+    client_dns:
+      - 1.1.1.1
+      - 1.0.0.1
+    client_allowed_ips:
+      - 0.0.0.0/0
+    max_devices_per_user: 10
+  authorization:
+    local_auth_enabled: true
+    disable_vpn_on_oidc_error: false
+  wireguard:
+    ipv4_masquerade_enabled: true
+  connectivity:
+    checks_enabled: true
+    checks_interval: 43200
+  other:
+    telemetry_enabled: false
+
 workload:
   main:
     podSpec:
       containers:
         main:
           probes:
             liveness:
-              enabled: false
-            readiness:
-              enabled: false
-            startup:
-              enabled: false
+              enabled: true
+              ## Set this to true if you wish to specify your own livenessProbe
+              custom: true
+              ## The spec field contains the values for the default livenessProbe.
+              ## If you selected custom: true, this field holds the definition of the livenessProbe.
+              spec:
+                exec:
+                  command:
+                    - /usr/bin/ash
+                    - bash
+                    - -c
+                    - curl --fail localhost:{{ .Values.service.main.ports.main.port }}/
+                failureThreshold: 5
+                initialDelaySeconds: 60
+                periodSeconds: 10
+                successThreshold: 1
+                timeoutSeconds: 10
```

Comment:
```suggestion
              enabled: true
              ## Set this to true if you wish to specify your own livenessProbe
              custom: true
              ## The spec field contains the values for the default livenessProbe.
              ## If you selected custom: true, this field holds the definition of the livenessProbe.
              spec:
                exec:
                  command:
                    - /usr/bin/ash
                    - bash
                    - -c
                    - curl --fail localhost:{{ .Values.service.main.ports.main.port }}/
                failureThreshold: 5
                initialDelaySeconds: 60
                periodSeconds: 10
                successThreshold: 1
                timeoutSeconds: 10
```
What is this? :D

Reply:
A) Custom is not a thing anymore
B) This is just an http probe?!

Reply:
its very weird that normal probes never work, was planning on use using the rest api curl method but this worked without issue, ill test  it again

Reply:
With this it just falled back to http.
But if you want to do something "weird" with curl, just use `type: exec`
and use the `command:` list to add whatever

Reply:
well no need .-. probes by default works now, tcp for startup has been meh lately.